tests/operators/test_docker_operator.py

```diff
@@ -57,6 +57,7 @@ def test_execute(self, client_class_mock, mkdtemp_mock):
 
         operator = DockerOperator(api_version='1.19', command='env', environment={'UNIT': 'TEST'},
                                   image='ubuntu:latest', network_mode='bridge', owner='unittest',
+                                  host_tmp_dir='/host/airflow',
```

Comment:
This is optional, but given that we've overridden the host temporary directory, it might make sense to change the mock on L46 to return something under /host/airflow and update the volume asserts on L79

Reply:
Makes sense to do this for consistency, yes.